.github/workflows/ci_frontend.yaml

```diff
@@ -0,0 +1,51 @@
+name: Frontend
+
+on:
+    workflow_dispatch: ~
+    workflow_call:
+        inputs:
+            type:
+                description: "Type of the build"
+                required: true
+                type: string
+
+jobs:
+    get-matrix:
+        runs-on: ubuntu-latest
+        name: "Get matrix"
+        outputs:
+            matrix: ${{ steps.matrix.outputs.prop }}
+        steps:
+            -   uses: actions/checkout@v3
+            -   name: "Get matrix"
+                id: matrix
+                uses: notiz-dev/github-action-json-property@release
+                with:
+                    path: '.github/workflows/matrix.json'
+                    prop_path: '${{ inputs.type }}.frontend'
+
+    frontend:
+        needs: get-matrix
+        runs-on: ubuntu-latest
+        name: "NodeJS ${{ matrix.node }}"
+        strategy:
+            fail-fast: false
+            matrix: ${{ fromJson(needs.get-matrix.outputs.matrix) }}
+
+        env:
+            APP_ENV: test_cached
+            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=8.0"
+
+        steps:
+            -
+                uses: actions/checkout@v3
+
+            -   name: Build application
+                uses: SyliusLabs/BuildTestAppAction@v2.0
+                with:
+                    build_type: "sylius"
+                    e2e_js: "yes"
+                    database_version: "8.0"
+                    php_version: "8.1"
+                    symfony_version: "6.4.*"
+                    node_version: ${{ matrix.node }}
```

Comment:
I just checked the build, can you update the action to cut it short?
I don't see a point in running migrations and fixtures when all we're checking for is whether the assets are getting built correctly

Reply:
IMO, the option `e2e_js` also start Chrome Headless and Symfony server, that are not required here. But, `e2e_js` is also required to run `yarn build` which is required.
It's not a blocking factor, but we may have to consider a change in the future.

Reply:
The whole workflow takes about 2 minutes. At the same time, we reuse the whole logic, so we keep the workflow as simple as possible. For me, 2m is not a big deal here.

Reply:
Fair enough